openmls/src/group/mls_group/test_mls_group.rs

```diff
@@ -1553,6 +1553,216 @@ fn update_group_context_with_unknown_extension<Provider: OpenMlsProvider + Defau
     );
 }
 
+// Test the successful update of Group Context Extension with type Extension::Unknown(0xff11)
+#[openmls_test]
+fn update_group_context_with_unknown_extension<Provider: OpenMlsProvider + Default>() {
```

Comment:
Is this test different from the one above with the same name? This fails to build because it has the same name.

Also, it doesn't seem to call `update_group_context_extensions`. I expect that it's supposed to test the new function?

Reply:
O hmm that was a bad merge on my part. I thought there might have been a test in the original PR https://github.com/xmtp/openmls/pull/22 but only see this one. @cameronvoell might you be able to write a test for this function here too?

Reply:
yep. I'll write up a test 👌